### ISSUE TITLE
on drop event not returning correct data type

### ISSUE THREAD

**Issue by Wealthyturtle** [SkriptLang/Skript#2151]
### Description

on drop event seems to not be returning an item stack, even though the event value is called `event-itemstack`

### Steps to Reproduce
```
function f(i: item) :: string:
    return name of {_i}

on drop:
    send f(event-itemstack)
```

### Expected Behavior

`event-itemstack` should be of data type item stack

### Errors / Screenshots

Line 5: The **1st argument given to the function 'f' is not of the required type item stack**. Check the correct order of the arguments and put lists into parentheses if appropriate (e.g. 'give(player, (iron ore and gold ore))'). Please note that storing the value in a variable and then using that variable as parameter will suppress this error, but it still won't work. (test.sk, line 5: send f(event-itemstack)')

### Server Information
* **Server version/platform:**       Spigot 1.14.2
* **Skript version:**                2.4.1-alpha3

**Comment by TheBentoBox** [comment]
There might be a workaround for this but it is at the very least misleading.

**Comment by Wealthyturtle** [comment]
Just realised this issue would be fixed by #2510
